# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/svgfe-subregion-bigger.yaml
# Commit: a0b3e2669a2d
# Full Hash: a0b3e2669a2dc49c467521cb9b7966ca05e3dbc3
# Author: Ashley Hale <ahale@mozilla.com>
# Date: 2024-05-08 09:49:23
# Regressor Bug: 1824502
# File Overlap Count: 1
# Description:
#   Bug 1824502 - Implement native SVG filter rendering in WebRender r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D209646
# ==============================================================================

diff -r d1df40ba7c88 -r a0b3e2669a2d gfx/wr/wrench/reftests/filters/svgfe-subregion-bigger.yaml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/wr/wrench/reftests/filters/svgfe-subregion-bigger.yaml	Wed May 08 04:38:26 2024 +0000
@@ -0,0 +1,35 @@
+---
+root:
+  items:
+    - type: stacking-context
+      bounds: [0, 50, 800, 800]
+      filters:
+        - svgfe: SourceGraphic
+        - svgfe: SourceAlpha
+        - svgfe: flood
+          linear: false
+          # this is expected to be at 10,60
+          subregion: [10, 10, 80, 100]
+          color: [255, 0, 0, 0.7]
+        - svgfe: flood
+          linear: false
+          # this is expected to be at 50,60
+          subregion: [50, 10, 80, 100]
+          color: [0, 255, 0, 0.7]
+        - svgfe: blendnormal
+          linear: false
+          # this is expected to be at 10,60
+          subregion: [10, 10, 160, 100]
+          in: 3
+          in2: 2
+        - svgfe: blendnormal
+          linear: false
+          # this is expected to be at 10,60
+          subregion: [10, 10, 160, 100]
+          in: 0
+          in2: 4
+      items:
+        - type: rect
+          # this is expected to be at 90,60
+          bounds: [90, 10, 80, 100]
+          color: [0, 0, 255, 0.7]